galaxychop/utils.py

```diff
@@ -57,15 +72,33 @@ def _get_rot_matrix(m, pos, vel, r_corte=None):
 
 
 def aling(m, pos, vel, r_corte):
-    """Esto devuelve las posiciones, las velocidades y J rotados, 
-    de forma que Jz quede alineado con z.
+    """This rotates the positions, speeds and angular 
```

Comment:
I'm not sure what you mean with "This rotates angular moments ". Maybe I don't understood the function. I think that we can ask to Vale, but if you are sure, is ok for me.